feature/bgp/policybase/otg_tests/link_bandwidth_test/README.md

```diff
@@ -0,0 +1,168 @@
+    named **'set_linkbw_0'** with the following `statements`
+    * statement[name='zero_linkbw']/
+      * actions/bgp-actions/set-ext-community/reference/config/ext-community-set-refs = 'linkbw_0'
+      * actions/bgp-actions/set-ext-community/config/options = ADD
+      * actions/bgp-actions/set-community/config/method = REFERENCE
+      * actions/config/policy-result = NEXT_STATEMENT
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * Create a `/routing-policy/policy-definitions/policy-definition/policy-definition`
+    named 'not_match_100_set_linkbw_1M' with the following `statements`
+    * statement[name='1-megabit-match']/
+      * conditions/bgp-conditions/match-community-set/config/community-set = 'regex_match_as100'
+      * conditions/bgp-conditions/match-community-set/config/match-set-options = INVERT
+      * actions/bgp-actions/set-ext-community/reference/config/ext-community-set-refs = 'linkbw_1M'
+      * actions/config/policy-result = NEXT_STATEMENT
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * Create a `/routing-policy/policy-definitions/policy-definition/policy-definition`
+    named 'match_100_set_linkbw_2G' with the following `statements`
+    * statement[name='2-gigabit-match']/
+      * conditions/bgp-conditions/match-community-set/config/community-set = 'regex_match_as100'
+      * conditions/bgp-conditions/match-community-set/config/match-set-options = ANY
+      * actions/bgp-actions/set-ext-community/reference/config/ext-community-set-refs = 'linkbw_2G'
+      * actions/config/policy-result = NEXT_STATEMENT
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * Create a `/routing-policy/policy-definitions/policy-definition/policy-definition`
+    named **'del_linkbw'** with the following `statements`
+    * statement[name='del_linkbw']/
+      * actions/bgp-actions/set-ext-community/config/options = 'REMOVE'
+      * actions/bgp-actions/set-ext-community/config/method = 'REFERENCE'
+      * actions/bgp-actions/set-ext-community/reference/config/ext-community-set-refs = 'linkbw_any'
+      * actions/config/policy-result = NEXT_STATEMENT
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * Create a `/routing-policy/policy-definitions/policy-definition/policy-definition`
+    named 'match_linkbw_0_remove_and_set_localpref_5' with the following `statements`
+    * statement[name='match_and_remove_linkbw_any_0']/
+      * conditions/bgp-conditions/match-ext-community-set/config/community-set = 'linkbw_any_0'
+      * actions/bgp-actions/set-ext-community/reference/config/ext-community-set-refs = 'linkbw_any_0'
+      * actions/bgp-actions/set-ext-community/config/options = 'REMOVE'
+      * actions/config/policy-result = NEXT_STATEMENT
+    * statement[name='set_LocPref_5']/
```

Comment:
```suggestion

```

The Link-bandwidth value 0 is interpreted by some implementation as weight "0" in WECMP group. In this implementations remaining members distribute traffic according to weights. 

Other implementations consider value 0 as "illegal" and consider route as not carryng link-bandwidth at all. This implementations creats ECMP group with all routes including this one, and ignores link-bandwidth of all members - distribute traffic equally.

This policy intention is to overcome this implementation difference, by deprefering (LocPref) routes with link-bandwidth 0 (only this routes) to prevent it become part of multipath, and remove  link-bandwidth comunity so route become clearly not "weight-ecmp-cpable".